app/assets/javascripts/webchat.js

```diff
@@ -0,0 +1,112 @@
+(function () {
```

Comment:
Should this be structured like https://github.com/alphagov/styleguides/blob/master/js.md#modules ?